README.md

```diff
@@ -154,8 +158,7 @@ To execute a TTG we must make it executable (this will declare the TTG  program
       tt->invoke();
 ```
 
-`ttg::execute()` must occur before, not after, sending any messages. Note also that we must ensure that only one such message must be generated. Since TTG execution uses the Single Program Multiple Data (SPMD) model,
-when launching the TTG program as multiple processes only the first process (rank) gets to send the message.
+`ttg::execute()` must occur before, not after, sending any messages. Note also that we must ensure that only one such message is generated. Since TTG execution uses the Single Program Multiple Data (SPMD) model, when launching the TTG program as multiple processes only the first process (rank) gets to send the message.
```

Comment:
Is it actually required that the first process (rank 0) initiates, or that `only one process (rank) gets to send the message`

Reply:
This text is not correct. Every process must call `ttg::execute()`.

Reply:
The "only some processes" referred to invoke, not execute. I rephrased.